### PR TITLE
FRONT-2786: Update bcl package.

### PR DESCRIPTION
**Pull request by tibi2303**

### CODE REVIEW

**Review by drishu — DISMISSED** [review]

**Review by Maxfire — DISMISSED** [review]
Checked and looks good to me.

File: kits/default/bcl-builder.config.js [hunk 1]
```diff
@@ -17,5 +17,6 @@ module.exports = {
         sourceMap: "file",
       },
     },
-  ]
+  ],
+  copy: []
```
Comment (escuriola) [inline kits/default/bcl-builder.config.js:21]: Only remark that at alexis branch there are more changes not included here:
image

Comment (drishu) [reply kits/default/bcl-builder.config.js:21]: that part of his branch is outdated

**Review by GilNovacomm — DISMISSED** [review]

File: kits/default/bcl-builder.config.js (hunk 1 above)
Comment (escuriola) [reply kits/default/bcl-builder.config.js:21]: I also have a warning on dropdown pattern, but the dropdown works. The rest of patterns seems to work ok.

image

**Review by drishu — DISMISSED** [review]

**Review by escuriola — DISMISSED** [review]

**Comment by escuriola** [comment]
Why is badge pattern here?

**Review by drishu — APPROVED** [review]

**Review by Maxfire — APPROVED** [review]
Approved 👍

**Review by escuriola — APPROVED** [review]

**Pull request merged**